part 2/src/component/card.css

```
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+}
+
+
+p {
+  font-size: 13px;
+  font-weight: 400px;
+  color: #666666;
+}
+
+.card {
+  width: 100%;
+  height: 334px;
+  border-radius: 15px;
+  box-shadow: 0px 5px 25px 0px #00000014;
```

0px은 단위가 생략 가능하고 box-shadow의 4번째 값이 0이면 생략 가능합니다! 

```suggestion
  box-shadow: 0 5px 25px #00000014;
```